src/assets/stylesheets/containers/search.sass

```
@@ -1,41 +1,74 @@
+.filter
+
+  &__caption
+    color: $color-gray
+    font-size: $font-size-sm
+    margin-bottom: $m-sm
+
+  &__selected-value
+    font:
+      size: $font-size-base
+      weight: $font-weight-bold
+    margin-right: $m-xs
+
+  &__value
+    align-items: center
+    display: flex
+    justify-content: center
+
 .search
-  margin:
-    top: $m-xxl * 3
-    left: $m-xxl * 2
-    right: $m-xxl * 2
-  display: flex
 
-  &__icon
-    background-color: rgba($color-white, 0.5)
-    padding: $m-lg
-    img
-      max-height: $m-xl
+  &__btn
+    background-color: $brand-primary
+    border: 0
+    display: block
+    font-weight: $font-weight-bold
+    padding:
+      bottom: $m-md
+      top: $m-md
+    width: 100%
 
   &__filters
+    align-items: center
     background-color: $color-white
-    flex: 1 0 auto
     color: $color-black
     display: flex
+    flex: 1 0 auto
     justify-content: space-around
-    align-items: center
-
-  &__btn
-    background-color: $color-violet
-    border: 0
+    margin-bottom: $m-md
     padding:
-      top: $m-xl
-      bottom: $m-xl
-      left:  $m-xxl * 2
-      right:  $m-xxl * 2
-    font-weight: bold
+      bottom: $m-lg
+      left: $m-md
+      right: $m-md
+      top: $m-lg
```

You can use shorthand:
```suggestion
      padding: $m-lg $m-md
```